Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Eel/ElasticSearchQueryBuilder.php

```diff
@@ -449,6 +468,28 @@ public function fulltext($searchWord) {
 	}
 
 	/**
+	 * Wrap search term matches found in the result for highlighting purposes. Only available for fulltext searches.
+	 *
+	 * @param integer $fragmentSize
+	 * @param integer $fragmentCount
+	 * @return QueryBuilderInterface
+	 */
+	public function highlight($fragmentSize = 150, $fragmentCount = 2) {
+		$this->request['_source'] = array('__fulltext');
```

Comment:
I do not yet understand why we need to load the _source as well? The "highlight" request works nevertheless, even if this line is removed.